packages/opentelemetry-node/lib/elastic-node-sdk.js

```diff
@@ -72,8 +68,27 @@ class ElasticNodeSDK extends NodeSDK {
         defaultConfig.instrumentations =
             opts.instrumentations || getInstrumentations();
 
-        // Default logs exporter.
-        // TODO: handle other protocols per OTEL_ exporter envvars (or get core NodeSDK to do it). Currently hardcoding to http/proto
+        // Protocols for exporters. Default is `http/proto`
+        const OTLPProtocolMap = {
```

Comment:
nit: on naming

```suggestion
        const otlpProtocolMap = {
```

or something like `exporterPkgNameFromEnvVar`.

Reply:
`exporterPkgNameFromEnvVar` it is